baselines/dom.generated.d.ts

```diff
@@ -14178,6 +14178,15 @@ interface PromiseRejectionEventInit extends EventInit {
     reason?: any;
 }
 
+interface EventListenerOptions {
+    capture: boolean;
+}
+
+interface AddEventListenerOptions extends EventListenerOptions {
+    passive: boolean;
```

Comment:
The fact that this type is a `dictionary` in the spec, meaning every property should be optional. Otherwise things like `{ once: true }` would not be accepted. Can you update? The same with `EventListenerOptions.capture`